manager/manager.go

```diff
@@ -769,11 +769,35 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// As we are proposing having multiple ways to detect a container (in addition to cgroups)
+// need the ability to delete
```

Comment:
done

// Enables overwriting an existing containerData/Handler object for a given containerName
// createContainer just returns if a given containerName has a handler already
// Ex: rkt handler will want to take priority over the raw handler, but the raw handler might be created first